test/powershell/SDK/Breakpoint.Tests.ps1

```diff
@@ -0,0 +1,101 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Please add tests for any errors thrown (although we can't test down level not supported, since we don't currently test that).

Reply:
I added tests for scenarios where you try to enable/disable/remove a breakpoint that no longer exists. I also flushed out the other tests a bit to have better coverage. That is committed and pushed already.

I'm not sure what tests I can add for "errors thrown". There are assertions for various things that the caller is expected to check (passing in `$null` as a command/variable/script name, passing in 0 for the `line` or `column` arguments to `SetLineBreakpoint`, etc.), but since those are assertions I cannot test for them using Pester, can I?